test/e2e/soak_namespace.go

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
In the previous review I asked that this file be renamed to namespace.go for consistency.